src/app/loadout-drawer/LoadoutDrawerDropTarget.tsx

```diff
@@ -49,7 +49,7 @@ export default function LoadoutDrawerDropTarget({
       canDrop: (i) =>
         itemCanBeInLoadout(i) &&
         (i.classType === DestinyClass.Unknown || classType === i.classType),
-      collect: (monitor) => ({ isOver: monitor.isOver() && monitor.canDrop() }),
+      collect: (monitor) => ({ isOver: monitor.isOver({ shallow: true }) && monitor.canDrop() }),
```

Comment:
This just ensures the outer drawer highlight doesn't happen when you are dropping a single item with the new functionality.